Makefile

```diff
@@ -89,8 +89,7 @@ docu_htmlnoheader: docu_htmlnoheaderclean docu_check
 docu_check:
 	./.travis/check_docs.sh
 
-findbugs:
-	mvn org.codehaus.mojo:findbugs-maven-plugin:check
+findbugs: $(SUBDIRS)
```

Comment:
I think a better way to do this would be to redefine the variables in the toplevel `Makefile` slightly:

```
JAVA_DIRS =api certificate-manager ...
SUBDIRS=$(JAVADIRS) ... and the others ...
```

And then define the top level `findbugs` target like this:
```
findbugs: $(JAVA_DIRS)
```

That would then save you from having to define a phony `findbugs` target in all the non-Java modules.

Reply:
I do not think this approach scales. If we do this, we should do the same also for Docker directories. But then it doesn't work anymore. So I think its better to do all of this the same way despite it meaning that you have to add findbugs to few more places as phony.

Reply:
Why do you say it then doesn't work any more?

Reply:
Because we will maintain a lot of lists ... list of dirs with docker images (which includes some Java modules), list of dirs with Java code (which includes some docker images), list of dirs for tasks run in all dirs.

Reply:
Yeah, that would probably work. But I would not do it in this PR. It changes how the building of the `docker_images` folder works etc. It should be a separate PR.